sdk/python/pulumi_kubernetes.egg-info/requires.txt

```diff
@@ -0,0 +1,5 @@
+parver>=0.2.1
+pulumi<3.0.0,>=2.0.0
+pyyaml<5.2,>=5.1
+requests<2.22.0,>=2.21.0
+semver>=2.8.1
```

Comment:
Is this getting pulled in from the common codegen? This list doesn't match what I see in the schema.

Reply:
Yeah I removed it a couple of things from the schema because it is automatically added by codegen. https://github.com/pulumi/pulumi/blob/master/pkg/codegen/python/gen.go#L843-L861